assets/wizards/popups/views/analytics/Chart.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies.
+ */
+import { format } from 'date-fns';
+import humanNumber from 'human-number';
+import {
+	ResponsiveContainer,
+	AreaChart,
+	Area,
+	CartesianGrid,
+	XAxis,
+	YAxis,
+	Tooltip,
+} from 'recharts';
+
+const Chart = ( { data } ) => (
+	<div className="pa__chart">
+		<ResponsiveContainer width={ '100%' } height={ 300 }>
+			<AreaChart data={ data } margin={ { top: 5, right: 20, bottom: 5, left: -15 } }>
+				<Area type="monotoneX" dataKey="value" stroke="#36f" fill="#3366ff1c" />
```

Comment:
Use variants of Newspack colors? https://github.com/Automattic/newspack-plugin/blob/master/assets/shared/scss/_colors.scss

Reply:
This is using the Newspack primary color (`#36f`), `#3366ff1c` is that color with transparency. SCSS variables can't be used here, as these have to be passed in JS, not CSS.